lib/dictBuilder/zdict.h

```diff
@@ -90,6 +91,17 @@ typedef struct {
     ZDICT_params_t zParams;
 } ZDICT_cover_params_t;
 
+typedef struct {
+    unsigned k;                  /* Segment size : constraint: 0 < k : Reasonable range [16, 2048+] */
+    unsigned d;                  /* dmer size : constraint: 0 < d <= k : Reasonable range [6, 16] */
+    unsigned f;                  /* log of size of frequency array : constraint: 0 < f <= 31 : 1 means default(18)*/
+    unsigned steps;              /* Number of steps : Only used for optimization : 0 means default (32) : Higher means more parameters checked */
+    unsigned nbThreads;          /* Number of threads : constraint: 0 < nbThreads : 1 means single-threaded : Only used for optimization : Ignored if ZSTD_MULTITHREAD is not defined */
+    double splitPoint;           /* Percentage of samples used for training: the first nbSamples * splitPoint samples will be used to training, the last nbSamples * (1 - splitPoint) samples will be used for testing, 0 means default (1.0), 1.0 when all samples are used for both training and testing */
+    unsigned finalize;           /* Percentage of training samples used for ZDICT_finalizeDictionary: 0 means default(100) */
+    unsigned skip;               /* Number of dmer skipped between each dmer counted in computeFrequency: constraint: 0 <= skip < k */
```

Comment:
This is getting to be a lot of parameters. Lets combine `skip` and `finalize` into a single parameter, called `skip`, or `acceleration`, etc.

We don't need to go into details of exactly what it does, just say higher means faster and less accurate, and 1 is the default (0 can clamp to 1). We'll use it internally to skip in `computeFrequency`, and decide on the number of samples passed to `ZSTD_finalizeDictionary()`.